backend/module/chaos/serverexperimentation/xds/ecds_faults_generator.go

```diff
@@ -171,6 +171,10 @@ func (g ECDSFaultsGenerator) createAbortDelayConfig(httpFaultConfig *serverexper
 	var abort *gcpFilterFault.FaultAbort
 	var delay *gcpFilterCommon.FaultDelay
 
+	pcntage, err := GetHTTPFaultPercentage(httpFaultConfig)
```

Comment:
nit: `percentage`, doesn't hurt to spell it out :)

Reply:
Ha you aint seen my keyboard - every character is a pain xD